R/duplicated_paste.R

```diff
@@ -0,0 +1,15 @@
+#' Detect duplicated strings
+#'
+#' @inheritDotParams base::paste
+#' @return  A logical vector of the same length as the longest vector passed to
+#'   `...`.
+#' @export
+#' @examples
+#' duplicated_paste(c("a", "a"), 1:2)
+#' paste(c("a", "a"), 1:2)
+#'
+#' duplicated_paste(c("a", "a"), c(1, 1))
+#' paste(c("a", "a"), c(1, 1))
```

Comment:
Examples are useful. On any package website, users can see what the function outputs even without running any code. For example:

https://2degreesinvesting.github.io/tilt.company.match/reference/abort_if_incomplete.html#ref-examples